internal/hcops/load_balancer.go

```diff
@@ -731,6 +741,11 @@ func (l *LoadBalancerOps) ReconcileHCLBTargets(
 				continue
 			}
 
+			if numberOfTargets >= lb.LoadBalancerType.MaxTargets {
+				klog.InfoS("cannot add ip target because max number of targets have been reached", "op", op, "service", svc.ObjectMeta.Name, "targetName", k8sNodeNames[int64(id)])
```

Comment:
Shouldn't this be a warning?

Reply:
`klog` does not have an interface for structured logging with the warning level, only for `InfoS` and `ErrorS`. We can make this error if you prefer that.

Currently this triggers the `ErrorCodeResourceLimitExceeded` check a few lines down, where we also log to `InfoS`.

Reply:
If the desired state if not reached, I assume something is wrong, so I'd go with an error. But feel free to keep it as is.